### DISCUSSION TITLE
Expand only one category at the time

### DISCUSSION THREAD

**Discussion by badiuciprian** [GrapesJS/grapesjs#5834] [Q&A]
Hi, is there a way to have only one trait category opened once? I have a lot of traits and many categories and I need a way to expand only one category at the time. The same for blocks. Thank's

**Comment by artf** [comment]
Currently, the only way right now would be to implement your own UI.

**Reply by badiuciprian** [reply]
Hi, thank you for the reply. How can I achieve that? using the Custom Trait Manager?

**Reply by artf** [reply]
Here you can find how to create your [custom Trait Manager UI](https://grapesjs.com/docs/modules/Traits.html#custom-trait-manager)

**Reply by badiuciprian** [reply]
ok, thank you. I will give it a try
